src/_guides/libraries/index.md

```diff
@@ -38,11 +38,11 @@ at the following links.
 ## Other resources
 
 API reference documentation
-: * [api.dartlang.org:]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}})
+: * [Dart API:]({{site.dart_api}}/{{site.data.pkg-vers.SDK.channel}})
```

Comment:
What do you think about using the macros on the LHS, as well as in the link? It's a little ugly because of the "https://", but we could work around that if necessary.